downstream/modules/platform/ref-pac-inputs-outputs.adoc

```diff
@@ -0,0 +1,50 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-08
+:_mod-docs-content-type: REFERENCE
+
+[id="pac-inputs-outputs_{context}"]
+= Policy enforcement inputs and outputs
+
+== Input data
```

Comment:
I would include the descriptive sentence under the primary heading and make the table caption "Input data"

.Input data
[cols="1,1,1",options="header"]
...